flare/ase/calculator.py

```diff
@@ -81,6 +82,51 @@ def get_forces_mgp(self, atoms):
         atoms.get_uncertainties = self.get_uncertainties
         return forces
 
+    def get_forces_mgp_par(self, atoms):
+        return self.get_forces_mgp_serial(atoms)
+#        comm = MPI.COMM_WORLD
```

Comment:
why all the MPI command is commented out? maybe we can leave a note in the pull request that this is yet to be tested.

Reply:
I think we might be able to use multiprocessing.shared_memory instead of mpi4py. I tested it, but there's some conflict inside of ASE's own parallel setting, calling QE and this MGP parallelization. 
The tricky part is we need to use `mpirun -n xx python **.py` if we use mpi4py, I think we must avoid this, and trying multiprocessing is safer. 

BTW, the `shared memory` feature of `multiprocessing` is new (only supported) in python 3.8....... We might need to transfer to this version....